### PR TITLE
Update README to specify tag requirements

### PR DESCRIPTION
**Pull request by MartinSnyder**
Hi Dale,

Didn't take me long to figure this out, but I thought I'd leave this breadcrumb for future users so they don't get confused about how to get sbt-dynver to recognize tags.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,16 @@ Other than that, as `sbt-dynver` is an AutoPlugin that is all that is required.
 | when there are no commits, or the project isn't a git repo           | HEAD+20140707-1030             | true       |
 ```
 
+## Tag Requirements
+
+In order to be recognized by sbt-dynver, tags must meet the following two requirements:
+1. Must be an annotated tag (as opposed to a lightweight tag)
```
Comment (dwijnand) [inline README.md]: The requirement that tags must be annotated/non-lightweight was actually lifted in [v1.1.0](https://github.com/dwijnand/sbt-dynver/releases/tag/v1.1.0), see #15. So can I ask you to remove this line please?

File: README.md [hunk 2]
```diff
@@ -41,6 +41,16 @@ Other than that, as `sbt-dynver` is an AutoPlugin that is all that is required.
...
+2. Must begin with the letter 'v'
```
Comment (dwijnand) [inline README.md]: Let's be really specific :) It must begin with the letter 'v' and a digit: `v[0-9]`.

File: README.md [hunk 3]
```diff
@@ -41,6 +41,16 @@ Other than that, as `sbt-dynver` is an AutoPlugin that is all that is required.
...
+
+If you're not seeing what you expect, then start with this:
+
+    git tag -af v0.0.1 -m "Initial version tag for sbt-dynver"
```
Comment (dwijnand) [inline README.md]: Real, real nitpick: could you drop the `-f` :)

Comment (MartinSnyder) [inline README.md]: I'll make these revisions, no problem. Thanks for the additional detail.

**Comment by dwijnand** [comment]
Thanks @MartinSnyder.

**Pull request merged**